api/envoy/config/route/v3/route_components.proto

```diff
@@ -1106,7 +1110,8 @@ message RouteAction {
   // should not be set if this field is used.
   google.protobuf.Any retry_policy_typed_config = 33;
 
-  // Indicates that the route has request mirroring policies.
+  // Specify a set of route request mirroring policies.
+  // Inherit from the request_mirror_policies of its virtual host if not specified.
```

Comment:
Can you comment on what happens if it is specified both here and in the upper level? Do they get merged or does one of them win over the other?

Reply:
That's a good point. The most internal policies win over the other. I add the comments. Thanks.